install/kubernetes/cilium/templates/hubble-ui/deployment.yaml

```diff
@@ -31,6 +51,10 @@ spec:
       securityContext:
         runAsUser: 1001
       {{- end }}
+      restartPolicy: Always
+{{- if and (or (and (eq .Release.Namespace "kube-system") (gt $k8sMinor "10")) (ge $k8sMinor "17") (gt $k8sMajor "1")) .Values.enableCriticalPriorityClass }}
+      priorityClassName: system-cluster-critical
```

Comment:
I don't think that Hubble UI should be marked as `system-cluster-critical`. If the cluster is under pressure, I think evicting Hubble UI should be OK as global visibility should still be provided by Hubble Relay that can be queried using the Hubble CLI.

Reply:
and `hubble-relay` as well